cg/apps/tb/api.py

```diff
@@ -154,7 +154,7 @@ def add_pending_analysis(
             "config_path": config_path,
             "out_dir": out_dir,
             "priority": priority,
-            "data_analysis": str(data_analysis).upper(),
+            "data_analysis": data_analysis.upper(),
```

Comment:
```suggestion
            "data_analysis": str(data_analysis).upper(),
```

Reply:
I must admit I am confused. I had this but then you wanted it to be pipeline. In the end you do want a string passed, correct?

Reply:
yes, otherwise it serializes it as <Pipeline.RSYNC> in the json file

Reply:
But it is better that it is a Pipeline all the way until its serialized. That way we know nothing that is not part of Pipeline should be passed as argument